### ISSUE TITLE
[Programmatic Access – Make Code - Advanced]State as 'Expand/Collapsed' is not announced by screen reader for 'Advanced' control.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3665]
User Experience:
Screen Reader users will get impacted if screen reader is not providing information about the state 'expand/Collapsed'  then user who rely on screen reader is not able to get the information about the control whether the state as 'expand/collapsed' then end users will get confused.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Open F12 Developer tool.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Functions" controls appearing under Advance section and select it.
6. Verify the issue.

Actual Result:
State is not defined for 'Set Hours' dropdown button. screen reader is not giving information about the state as 'Collapsed/Expand' it stays silent at that time when user activate the control

Expected Result:
State should be defined for control. Screen reader should announce the state as 'Expand/Collapsed' when it gets activate.
Eg. aria-expand='True' should be used.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[WCAG 4.1.2 – Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html%7D)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

[Mas4.1.2_State_Advanced button.zip](https://github.com/microsoft/pxt-microbit/files/5528885/Mas4.1.2_State_Advanced.button.zip)

![Uploading Advanced.png…]()
